.Gruntfile.babel.js

```diff
@@ -115,7 +115,7 @@ module.exports = (grunt) => {
         cmd: './node_modules/.bin/mocha --require Gruntfile.js -R spec --bail',
         options: {env: {LOAD_NO_FILE: 'true', ...process.env}}
       },
-      standard: './node_modules/.bin/standard "**/*.{js,vue}"',
+      standard: './node_modules/.bin/standard "**/*.{js,vue}" --fix',
```

Comment:
`--fix` should only be called intentionally by the developer. If it is placed here it will be run every time a file changes, and that's not good.